06-building-a-program-start-to-finish/setting-up-software-development-environments-(deployment)/output/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Heroku
+- **Deliver**: live URLs to a `review` app, your `staging` & `production` apps
+- **With**: *independently*
+
+#### Targeted Learning Outcomes
+- [review app](https://inverted-index-repo-jed--pr-23.herokuapp.com/)
+- [staging app](https://inverted-index-jed-staging.herokuapp.com/)
+- [production app](https://inverted-index-jed-production.herokuapp.com/)
```

Comment:
Thank you for pointing that out, this is because my master branch is currently empty, I intend to have it up and running, once i am given the go-ahead to merge my staging branch to the master branch.